hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -13,6 +13,160 @@
                      "hw/ip/tlul/data/tlul_testplan.hjson"]
 
   testpoints: [
+
+    //////////////////
+    // Host Device  //
+    // RV_CORE_IBEX //
+    //////////////////
+    //
+    // Note: the Debug Module will be referred to as RVDM.
+    {
+      name: chip_ibex_debug_basic
+      desc: '''Verify correct operation of rv_core_ibex while in debug mode.
+
+            Trigger the RVDM to send a debug request to rv_core_ibex, which should then jump to the
+            debug ROM and execute the code there before returning via `dret` and executing the rest
+            of the main program.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_debug_single_step_test
+      desc: '''Verify correct operation of rv_core_ibex while single-step debugging.
+
+            Set `dcsr.step` in the core.
+            Trigger the RVDM to send a debug request to rv_core_ibex, which should then jump to the
+            debug ROM and start code execution.
+            Write `dmcontrol.resumereq`, and check that the core jumps out of debug mode, executes
+            one instruction, then jumps back into debug mode.
+            Do this a random number of times before returning via `dret`and executing the rest of
+            the main program.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_debug_ebreak_test
```

Comment:
not sure we need to test this instruction at the system level if it is already tested at block level.

Reply:
@andreaskurth can you take this one into account when updating existing tetspoints such as [chip_rv_dm_perform_debug](https://www.google.com/url?q=https://github.com/lowRISC/opentitan/blob/6f7094c7b1fc37b0a1955a04d3c259aab86d50c5/hw/top_earlgrey/data/chip_testplan.hjson%23L548&sa=D&source=docs&ust=1663272181668386&usg=AOvVaw3tIZfeqvAfzelQUevBPR1_).